models/courier/Courier.validation.js

```diff
@@ -0,0 +1,21 @@
+import Joi from 'joi';
+import { vehicleType } from '../../constants/vehicleType';
+import { addressValidationSchema, phoneNumberPattern } from '../../helpers/validation';
+import { accountStatus } from '../../constants/accountStatus';
+
+const CourierValidationSchema = Joi.object({
+  userId: Joi.required(),
```

Comment:
any type?

Reply:
set value type here, please, otherwise okay

Reply:
![image](https://github.com/Autumn-fairytale/AFT-back/assets/128153859/ac9bd48f-5945-411d-b104-0d0dd063ae8c)

Sorry, but I've already done that

Reply:
my bad, opened wrong commit